### PR TITLE
Enabling dynamic obstacle detection

### PR DESCRIPTION
**Pull request by orduno**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | #544 |
| Primary OS tested on | Ubuntu |
| Robotic platform tested on | gazebo simulation of turtlebot |

---

## Description of contribution in a few bullet points

* Some important code was commented out in the obstacle layer by mistake
* Changed params to use obstacle layer by default.

### CODE REVIEW

**Comment by crdelsey** [comment]
The defect was introduced in #387

File: nav2_bringup/launch/nav2_params.yaml [hunk 1]
```diff
@@ -22,21 +22,23 @@ local_costmap:
   local_costmap:
     ros__parameters:
       obstacle_layer:
-        enabled: False
+        enabled: True
```
Comment (crdelsey) [inline nav2_bringup/launch/nav2_params.yaml]: I'd prefer to leave the obstacle layer off by default unless you've run the system test enough times to ensure the success rate hasn't gone down by doing this.

Comment (orduno) [reply nav2_bringup/launch/nav2_params.yaml]: OK, I've changed it

**Comment by orduno** [comment]
@crdelsey @mkhansen-intel I've disabled the obstacle layer by default as requested.

**Pull request merged**